docs/credentials/credential-sharing.md

```diff
@@ -35,3 +36,4 @@ To unshare a credential:
 3. Select **Sharing**.
 4. Select **Options** <span class="inline-image">![Options menu icon](/_images/common-icons/three-dot-options-menu.png){.off-glb}</span> on the user you want to remove.
```

Comment:
And finally, realised we also need update removing steps slight to:

4. Find the user(s) or projects(s) you want to remove.
5. Click the trash icon.
6. Save.

Reply:
I updated the instructions to match the latest UI on the testing instance, let me know if you notice any inconsistencies!